src/Storages/StorageReplicatedMergeTree.h

```diff
@@ -230,6 +230,9 @@ class StorageReplicatedMergeTree final : public MergeTreeData
     static void dropReplica(zkutil::ZooKeeperPtr zookeeper, const String & zookeeper_path, const String & replica,
                             Poco::Logger * logger, MergeTreeSettingsPtr table_settings = nullptr, std::optional<bool> * has_metadata_out = nullptr);
 
+    void dropReplica(const String & drop_zookeeper_path, const String & drop_replica,
+                            Poco::Logger * logger, MergeTreeSettingsPtr table_settings = nullptr, std::optional<bool> * has_metadata_out = nullptr);
```

Comment:
You don't pass `table_settings` and `has_metadata_out`, so consider removing these arguments

Reply:
ok.

Reply:
https://github.com/ClickHouse/ClickHouse/blob/b0da0cffe75a6570288ed5180ca9f9b68168f6d6/src/Storages/StorageReplicatedMergeTree.h#L233